src/openqaoa-braket/backends/gates_braket.py

```diff
@@ -45,49 +48,38 @@ def apply_2q_rotation_gate(
         qubit_1: int,
         qubit_2: int,
         rotation_object: RotationAngle,
-        circuit: Circuit
+        circuit: Circuit,
     ) -> Circuit:
-        circuit += braket_gate(
-            qubit_1, 
-            qubit_2, 
-            rotation_object.rotation_angle
-        )
+        circuit += braket_gate(qubit_1, qubit_2, rotation_object.rotation_angle)
         return circuit
 
     @staticmethod
-    def apply_1q_fixed_gate(
-        braket_gate,
-        qubit_1: int,
-        circuit: Circuit
-    ) -> Circuit:
+    def apply_1q_fixed_gate(braket_gate, qubit_1: int, circuit: Circuit) -> Circuit:
         circuit += braket_gate(qubit_1)
         return circuit
 
     @staticmethod
     def apply_2q_fixed_gate(
-        braket_gate,
-        qubit_1: int,
-        qubit_2: int,
-        circuit: Circuit
+        braket_gate, qubit_1: int, qubit_2: int, circuit: Circuit
     ) -> Circuit:
         circuit += braket_gate(qubit_1, qubit_2)
         return circuit
 
     def apply_gate(self, gate: gates_core.Gate, *args) -> Circuit:
         selected_braket_gate = self.gate_selector(gate)
         if gate.n_qubits == 1:
-            if hasattr(gate, 'rotation_object'):
+            if hasattr(gate, "rotation_object"):
                 # *args must be of the following format -- (qubit_1,rotation_object,circuit)
                 return self.apply_1q_rotation_gate(selected_braket_gate, *args)
             else:
                 # *args must be of the following format -- (qubit_1,circuit)
                 return self.apply_1q_fixed_gate(selected_braket_gate, *args)
         elif gate.n_qubits == 2:
-            if hasattr(gate, 'rotation_object'):
+            if hasattr(gate, "rotation_object"):
                 # *args must be of the following format -- (qubit_1,qubit_2,rotation_object,circuit)
                 return self.apply_2q_rotation_gate(selected_braket_gate, *args)
             else:
                 # *args must be of the following format -- (qubit_1,qubit_2,circuit)
                 return self.apply_2q_fixed_gate(selected_braket_gate, *args)
         else:
-            raise ValueError("Only 1 and 2-qubit gates are supported.")
\ No newline at end of file
+            raise ValueError("Only 1 and 2-qubit gates are supported.")
```

Comment:
Maybe it makes sense to be more explicit (the code fail also when n_qubits is none):
```
raise ValueError(f"Only 1 and 2-qubit gates are supported. Instead, I received n_qubits={n_qubits}")
```

Reply:
Sure, but that's not directly relevant for this PR.

Reply:
Isn't this new code?

Reply:
we can fix this separately in another PR.